source-mysql/replication.go

```diff
@@ -148,6 +150,7 @@ type mysqlReplicationStream struct {
 	errCh  chan error         // Error output channel for the replication worker goroutine
 
 	gtidTimestamp time.Time // The OriginalCommitTimestamp value of the last GTID Event
+	gtidString    string    // The GTID value of the last GTID event, formatted as a "<uuid>:<counter>" string.
```

Comment:
The `mysqlSourceInfo` mentions that this is only set if GTIDs are enabled. If GTIDs are not enabled, does this end up in the output document as an empty string, or is it completely omitted?

Reply:
Currently it will be completely omitted if GTIDs are disabled.